rfc-drafts/stream.md

```diff
@@ -124,6 +124,52 @@ where
 }
 ```
 
+## Next method/struct
+
+We should also implement a next method, similar to [the implementation in the futures crate](https://docs.rs/futures-util/0.3.5/src/futures_util/stream/stream/next.rs.html#10-12).
+
+```rust
+/// Future for the [`next`](super::StreamExt::next) method.
```

Comment:
This comment appears to be a copy-paste artifact. The `Iterator` adapters have a really nice comment scheme that I think we can adopt for `Stream` as well.

```suggestion
/// A future that advances the stream and returns the next value.
///
/// This `struct` is created by the [`next`] method on [`Stream`]. See its
/// documentation for more.
///
/// [`next`]: trait.Stream.html#method.next
/// [`Stream`]: trait.Stream.html
```

Reply:
By the way, originally `futures-util` used that style, but we changed it to the current simple style because:

* https://github.com/rust-lang/futures-rs/issues/1214:
  > DRY (do not repeat yourself): Let's _not_ include a short description. The combinator method always already has a more detailed, better-phrased description. Linking to that is enough/better.
* It needs to click the combinator type docs one by one to see which combinator method returns that type.

(I do not mean to strongly recommend the style of futures, but I don't think std's style is necessarily great.)